lib/include/hse_ikvdb/kvset_view.h

```diff
@@ -90,7 +90,7 @@ struct kvset_view {
     struct kvset *kvset;
     uint64_t      nodeid;
     uint          eklen;
-    char          ekbuf[20];
+    char          ekbuf[28];
```

Comment:
I feel like this changes a bit with each PR.

Reply:
That is because it does :)

Reply:
Sorry!  It does :-)  I needed a bit more space to make gcc stop complaining about an snprintf() "overflow" (grrr, isnt' that the point of specifying the clip size???).